desc/equilibrium/equilibrium.py

```diff
@@ -1297,6 +1297,51 @@ def Zb_lmn(self):
     def Zb_lmn(self, Zb_lmn):
         self.surface.Z_lmn = Zb_lmn
 
+    @optimizable_parameter
+    @property
+    def I(self):  # noqa: E743
+        """float: Net current linking the plasma surface toroidally."""
```

Comment:
Maybe specify that it is a surface current? as `I` itself is something that can be calculated for interior surfaces too